third_party/tensorrt/tensorrt_oss.BUILD

```diff
@@ -0,0 +1,60 @@
+# NVIDIA TensorRT Open Source Plugins
+# This package contains build targets for select TensorRT plugins included in the
+# TensorRT open source repository.
+load("@local_config_cuda//cuda:build_defs.bzl", "cuda_default_copts", "cuda_library")
+
+exports_files(["LICENSE"])
+
+cuda_library(
+    name = "plugin_common",
+    srcs = [
+        "plugin/common/kernels/common.cu.cc",
+    ],
+    hdrs = [
+        "plugin/common/bboxUtils.h",
+        "plugin/common/checkMacrosPlugin.h",
+        "plugin/common/plugin.h",
+    ],
+    strip_include_prefix = "plugin/common",
+    deps = [
+        "@local_config_tensorrt//:tensorrt",
+        "@local_config_tensorrt//:tensorrt_headers",
+    ],
+)
+
+cc_library(
+    name = "nms_plugin_hdrs",
+    hdrs = [
+        "plugin/efficientNMSPlugin/efficientNMSInference.h",
+        "plugin/efficientNMSPlugin/efficientNMSParameters.h",
+        "plugin/efficientNMSPlugin/efficientNMSPlugin.h",
+    ],
+    strip_include_prefix = "plugin/efficientNMSPlugin",
+)
+
+cuda_library(
+    name = "oss_plugins",
```

Comment:
The build file inside google uses this name: nvinfer_plugin_nms
Please s/oss_plugins/nvinfer_plugin_nms/

Also, inside google, the plugin source is under
third_party/tensorrt/plugin,
and the build file for building the plugin is under
third_party/tensorrt/plugin/BUILD

your PR here put the build file under third_party/tensorrt/BUILD
Can you change this PR so that builds are from the plugin subdirectory?

This will affect how the library is used.
for example, within google we use third_party/tensorrt/plugin:nvinfer_plugin_nms
but with your current PR, we will use third_party/tensorrt:nvinfer_plugin_nms in OSS

after the change requested here, this PR should be good for merging.

Reply:
Ok what do you have for "include_prefix" parameter here? I'm changing it to `third_party/tensorrt/nvinfer_plugin_nms`. Is this correct?

Reply:
Also do I need to update `tensorflow/opensource_only.files` or should I remove those changes?

Reply:
The OSS build and google build only share one thing in comment, that is, how the TF-TRT build file refers to the plugin targets. The google build assumes the plugin targets are under third_party/tensorrt/plugin -- which is the only constraint for this PR, the rest is whatever that need to make it work in the OSS build.

Reply:
> This will affect how the library is used.
for example, within google we use third_party/tensorrt/plugin:nvinfer_plugin_nms
but with your current PR, we will use third_party/tensorrt:nvinfer_plugin_nms in OSS

Some confusion here. Remember there is a level of indirection via `tensorrt_configure.bzl` which creates the repository `@local_config_tensorrt` (at least in the OSS build, IDK how yours works). That makes the dependency name `@local_config_tensorrt//:nvinfer_plugin_nms` after I updated the files. 

I've pushed up a second commit which shows how I've tested this